application/templates/cms/create_chart_2.html

```diff
@@ -323,45 +352,26 @@ <h2 class="heading-medium">4. Panel chart options</h2>
                     </form>
                 </div>
 
-                {#
-                    PANEL LINE CHARTS _______________________________
-                    Are not more complex than regular line charts
-                    Must have ETHNICITY as their panels - we only need to set a column for the x-axis
-                #}
-                <div id="panel_line_chart_options" class="chart-builder-section chart-option-group grid-row"
-                     style="display:none">
-                    <h2 class="heading-medium">4. Panel line chart options</h2>
-                    <form>
-
-                        <div>
-                            <label for="panel-line__x-axis_column">X-Axis</label>
-                            <select id="panel-line__x-axis_column" class="form-control">
-                                <option>Time</option>
-                            </select>
-                        </div>
-                    </form>
-                </div>
-
                 <div id="chart_format_options" class="chart-builder-section chart-option-group grid-row"
                      style="display:none">
-                    <h2 class="heading-medium">5. Chart format options</h2>
+                    <h2 class="heading-medium builder-section-number">Chart format options</h2>
                     <form>
                         <div class="form-group">
                             <label for="number_format">Number format</label>
                             <select id="number_format" class="form-control"
                                     {% if form_disabled %}disabled="disabled"{% endif %}>
-                                <option value="none" selected>[None]</option>
-                                <option value="percent">Percentage</option>
+                                <option value="none" selected>None</option>
+                                <option value="percent">Percentage (%)</option>
                                 <option value="other">Other</option>
                             </select>
                             <div id="other_number_format" class="form-group" style="display:none">
-                                <label for="number_format_prefix">Prefix</label>
+                                <label for="number_format_prefix">Prefix (for example, £)</label>
```

Comment:
Should we have unicode escape sequence for the pound sign? Or we think it should be fine like this?

Reply:
https://stackoverflow.com/questions/13306693/should-i-encode-html-special-characters-when-using-utf-8

Apparently not?

Reply:
Apparently I'm just an old fogey.

Reply:
Yeah, way easier to read without. The only things you need to encode are `<` and `>` I think.